README.md

```diff
@@ -68,11 +68,11 @@ This provider's versions are able to work on the following versions of Apache Cl
 
 ## Operating system images
 
-Note: Cluster API Provider Apache CloudStack relies on a few prerequisites which have to be already
+Note: Cluster API Provider CloudStack relies on a few prerequisites which have to be already
 installed in the used operating system images, e.g. a container runtime, kubelet, kubeadm, etc.
 Reference images can be found in [kubernetes-sigs/image-builder][image-builder].
 
-Prebuilt images can be found [here](prebuilt-images)
+Prebuilt images can be found [here][prebuilt-images]
```

Comment:
How does markdown handle brackets back to pack? I always thought links required brackets followed by parentheses, so I'm not sure how to interpret this pattern

Reply:
Parenthesis are for inline links, brackets for reference links 
https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet#links

You can build the book locally and view the final result
```
make -C docs/book/ serve
```